src/mongocxx/write_concern.hpp

```diff
@@ -23,6 +23,7 @@
 
 #include <bsoncxx/stdx/optional.hpp>
 #include <bsoncxx/stdx/string_view.hpp>
+
```

Comment:
Is this change following style elsewhere? I don't remember that we separated the blocks for bsoncxx and mongocxx headers as a general principle, I think we just sorted them together,